src/mlj/interface.jl

```diff
@@ -73,20 +74,24 @@ function MMI.predict(m::Union{CLF_MODELS...}, (θ, c, classes), Xnew)
     return [MMI.UnivariateFinite(classes, preds[i, :]) for i in 1:size(Xmatrix,1)]
 end
 
-function MMI.fitted_params(m::Union{CLF_MODELS...}, (θ, c, classes))
+function MMI.fitted_params(m::Union{CLF_MODELS...}, (θ, features, c, classes))
     if c > 1
+        W = reshape(θ, :, c)
         if m.fit_intercept
-            W = reshape(θ, div(length(θ), c), c)
-            return (coefs = W, intercept = nothing)
+            return (coefs = coef_dict(W, features), intercept = W[end, :])
         end
-        W = reshape(θ, p+1, c)
-        return (coefs = W[1:p, :], intercept = W[end, :])
+        return (coefs = coef_dict(W[1:end-1,:], features), intercept = nothing)
```

Comment:
The previous implementation actually errors in this pass (`p` is undefined). Let me know I introduce another error :P